### PR TITLE
F2F-152 -Backend stack name ref updated

### PR DESCRIPTION
**Pull request by maruf-ahmedd**
### What changed
Added new backend stack ref in deploy.sh

### Why did it change
So frontend and backend stack-name are align

### Issue tracking
- [F2F-152](https://govukverify.atlassian.net/browse/F2F-152)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[F2F-152]: https://govukverify.atlassian.net/browse/F2F-152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -117,6 +117,15 @@
         "line_number": 20
       }
     ],
+    "infrastructure/lambda/public-api.yaml": [
```
Comment (richa-misra) [inline .secrets.baseline:120]: did we need to add any secrets for this?

File: deploy.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ stack_name="$1"
 
 if [ -z "$stack_name" ]
 then
-echo "😱 stack name expected as first argument, e.g. ./deploy di-ipv-cri-cic"
+echo "😱 stack name expected as first argument, e.g. ./deploy cic-cri-api-v1"
```
Comment (richa-misra) [inline deploy.sh:8]: could this be modified to `./deploy.sh cic-cri-api-v1` pls

File: .secrets.baseline (hunk 1 above)
Comment (maruf-ahmedd) [reply .secrets.baseline:120]: i have added the secret to reflect the new name in template file. Public api is using token from credential issuer.

File: deploy.sh (hunk 2 above)
Comment (maruf-ahmedd) [reply deploy.sh:8]: Sure..my understanding was this command is not being run as you can see its e.g.  Its for when you are deploying manually. Should i still change it?

File: .secrets.baseline (hunk 1 above)
Comment (richa-misra) [reply .secrets.baseline:120]: .secrets.baseline is getting generated by running `detect-secrets scan` all good

**Review by richa-misra — APPROVED** [review]

**Pull request merged**